andy/src/test/resources/codechecker/fixtures/MockitoWhenCalls.java

```diff
@@ -13,12 +15,14 @@ public class MockitoWhenCalls {
     void t1() {
         List<String> mockedList = mock(List.class);
 
+        Mockito.doThrow(IllegalAccessError.class).when(mockedList).toString();
```

Comment:
Can you not change this fixture, but add a new one instead? This way we are pretty sure existing behavior is kept!

Reply:
I should have done this from the beginning. I restored their state and added additional tests